src/twisted/newsfragments/10057.doc

```diff
@@ -0,0 +1 @@
+Fix a few dozen broken links to API documentation pages.
```

Comment:
Maybe. "The documentation was updated to use the Sphinx intersphinx extension for linking between our narrative documentation and the Twisted own API documentation."

Reply:
Your suggestion is a much better description of the PR, but the news fragments are for the release notes and I figured that the only thing the user is going to notice is that there are fewer broken links.